### PR TITLE
Doc: remove the 'Proposed' mark for the about to be recognized L2s

### PR DESCRIPTION
**Pull request by TheDZhon**
06aca6a2-c59c-4aa0-9542-bedc3e7008dd

- [x] [wstETH on zkSync Era](https://snapshot.org/#/lido-snapshot.eth/proposal/0xd6c4a71c36bef27c4b5997223bd8612fe19177b46b238e78802a4a27fd5cdc9e)
- [x] [wstETH on Mantle](https://snapshot.org/#/lido-snapshot.eth/proposal/0x349fa7409a99683405e71ddebaf5068f3dee7d4e6c9e4375198c4dc10c899bb9)
- [x] [wstETH on Linea](https://snapshot.org/#/lido-snapshot.eth/proposal/0x2e0a975282d741aaedc7bd80cee24986da67671e84d9cf9fb10da527ef5d1177)

---
:warning: Please don't merge yet while the snapshot vote haven't being finished

### CODE REVIEW

File: docs/deployed-contracts/index.md [hunk 1]
```diff
@@ -274,53 +274,53 @@ pagination_next: deployed-contracts/goerli
...
-  - ZkSyncBridgeExecutor
-  - ERC20BridgedUpgradeable
+- ZkSyncBridgeExecutor: [`0x139EE25DCad405d2a038E7A67f9ffdbf0f573f3c`](https://explorer.zksync.io/address/0x139EE25DCad405d2a038E7A67f9ffdbf0f573f3c) (proxy)
+- ZkSyncBridgeExecutor: [`0x13f46b59067f064c634fb17e207ed203916dccc8`](https://explorer.zksync.io/address/0x13f46b59067f064c634fb17e207ed203916dccc8) (impl)
+- L2ERC20Bridge: [`0xE1D6A50E7101c8f8db77352897Ee3f1AC53f782B`](https://explorer.zksync.io/address/0xE1D6A50E7101c8f8db77352897Ee3f1AC53f782B) (proxy)
+- L2ERC20Bridge: [`0x64Ee90B086c99fD3439354f382Fef25229A01F02`](https://explorer.zksync.io/address/0x64Ee90B086c99fD3439354f382Fef25229A01F02) (impl)
+- ERC20BridgedUpgradeable: [`0x703b52F2b28fEbcB60E1372858AF5b18849FE867`](https://explorer.zksync.io/address/0x703b52F2b28fEbcB60E1372858AF5b18849FE867) (proxy)
+- ERC20BridgedUpgradeable: [`0xc7a0daa1b8fea68532b6425d0e156088b0d2ab2c`](https://explorer.zksync.io/address/0xc7a0daa1b8fea68532b6425d0e156088b0d2ab2c) (impl)
+- ProxyAdmin: [`0xbd80e505ecc49bae2cc86094a78fa0e2db28b52a`](https://explorer.zksync.io/address/0xbd80e505ecc49bae2cc86094a78fa0e2db28b52a) for
+- ZkSyncBridgeExecutor
```
Comment (arwer13) [inline docs/deployed-contracts/index.md:291]:
```suggestion
  - ZkSyncBridgeExecutor
```

File: docs/deployed-contracts/index.md [hunk 2]
```diff
@@ -274,53 +274,53 @@ pagination_next: deployed-contracts/goerli
...
+- ERC20BridgedUpgradeable
```
Comment (arwer13) [inline docs/deployed-contracts/index.md:292]:
```suggestion
  - ERC20BridgedUpgradeable
```

File: docs/deployed-contracts/goerli.md [hunk 3]
```diff
@@ -167,50 +167,50 @@ Note that there will be no comprehensive Lido testnet environments available for
...
-  - ERC20BridgedUpgradeable
+- ZkSyncBridgeExecutor: [`0x4097BC99a9B5fB253B938d105111FFcb862Be668`](https://goerli.explorer.zksync.io/address/0x4097BC99a9B5fB253B938d105111FFcb862Be668) (proxy)
+- ZkSyncBridgeExecutor: [`0x213ec81bB9740c75d2BC186378cf734F9306e7e5`](https://goerli.explorer.zksync.io/address/0x213ec81bB9740c75d2BC186378cf734F9306e7e5) (impl)
+- L2ERC20Bridge: [`0x0E1E4af13Bb95C84275b52b6edE1cb5F8c11Abe7`](https://goerli.explorer.zksync.io/address/0x0E1E4af13Bb95C84275b52b6edE1cb5F8c11Abe7) (proxy)
+- L2ERC20Bridge: [`0xfD744ba1867B6e6da437bce3ba44f5898A72B9c3`](https://goerli.explorer.zksync.io/address/0xfD744ba1867B6e6da437bce3ba44f5898A72B9c3) (impl)
+- ERC20BridgedUpgradeable: [`0x6750967d6e4712Ae6cF9401a87dc34642339E0Ef`](https://goerli.explorer.zksync.io/address/0x6750967d6e4712Ae6cF9401a87dc34642339E0Ef) (proxy)
+- ERC20BridgedUpgradeable: [`0x75e79BCe0F5a81b6D0097CCB473D867940A4C2Ca`](https://goerli.explorer.zksync.io/address/0x75e79BCe0F5a81b6D0097CCB473D867940A4C2Ca) (impl)
+- ProxyAdmin: [`0xe7e7befca267e5967e20d135941535a2396ccdf4`](https://goerli.explorer.zksync.io/address/0xe7e7befca267e5967e20d135941535a2396ccdf4) for
+- ZkSyncBridgeExecutor
```
Comment (arwer13) [inline docs/deployed-contracts/goerli.md:184]:
```suggestion
  - ZkSyncBridgeExecutor
```

File: docs/deployed-contracts/goerli.md [hunk 4]
```diff
@@ -167,50 +167,50 @@ Note that there will be no comprehensive Lido testnet environments available for
...
+- ERC20BridgedUpgradeable
```
Comment (arwer13) [inline docs/deployed-contracts/goerli.md:185]:
```suggestion
  - ERC20BridgedUpgradeable
```

**Review by arwer13 — CHANGES_REQUESTED** [review]

**Review by arwer13 — APPROVED** [review]

**Review by grstepanov — APPROVED** [review]
lgtm, it is ready to get merged as soon as the bridges get recognized-recognized

**Pull request merged**